src/global.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '@asyncapi/converter';
```

Comment:
```suggestion
declare module '@asyncapi/converter' {
  export async function convert(spec: string, targetVersion: string, options: any): Promise<any>
}
```

@peter-rr try writing the module as per your needs, I am just giving an example for `convert` do this for `@asyncapi/spec` as well. 

You can take some inspiration from https://github.com/asyncapi/server-api/blob/master/src/server-api.d.ts